client/apps/edit/components/content/sections/text/index.coffee

```diff
@@ -23,7 +23,7 @@ window.process = {env: {NODE_ENV: sd.NODE_ENV}}
   setSelectionToStart,
   stripGoogleStyles } = require '../../../../../../components/rich_text/utils/index.coffee'
 editor = (props) -> React.createElement Editor, props
-{ div, nav, a, span, p, h3 } = React.DOM
+{ div, nav, a, span, p, h3, h2 } = React.DOM
```

Comment:
Curious why we include h2 here when it doesn't seem like it's being used in the file? I looked for h3 as well but it doesn't seem to be there either - maybe i'm missing some context with draft tho?